### PR TITLE
set ansible playbook environment proxy settings for no proxy

### LINKED ISSUE #384: Bootstrap step YorcOnDemandLocationResources fails on setup with http proxy defined
# Bug Report

## Description

On a setup having external access through a proxy, a bootstrap fails at step YorcOnDemandLocationResources when it attempts to connect to the Alien4Cloud UI,
as it tries to connect going through the proxy, even if environment variables NO_PROXY and no_proxy are set correctly.

## Output of `yorc version`

3.1.2

Affetcs as well 3.2.0 snapshots

## Priority

Medium

* High = We should stop anything we are doing and do this. If you're writing this calmly, you should probably choose another option. If it's really that urgent, please make sure we read this.
* Medium = It's important that we do this within a few days.
* Low = We will consider this on our next sprint planning.

(Please be aware that your priority may not match ours, we'll use this as guidance only).

### PR DESCRIPTION
**Pull request by stebenoist**
remove defers from loops

# Pull Request description

## Description of the change
Empty proxy setting in playbook environment for all tasks for Alien OnDemand resources location creation script in bootstrap type as it's done for orchestrator creation : we assume that Alien is accessible without proxies.

### Description for the changelog
* Bootstrap step YorcOnDemandLocationResources fails on setup with http proxy defined ([GH-384](https://github.com/ystia/yorc/issues/384))

## Applicable Issues

Fixes #384

### CODE REVIEW

File: commands/bootstrap/resources.go [hunk 1]
```diff
@@ -86,7 +86,6 @@ func extractResources(resourcesZipFilePath, resourcesDir string) error {
 			if err != nil {
 				return err
 			}
-			defer dest.Close()
```
Comment (loicalbertin) [inline commands/bootstrap/resources.go]: Why did you removed the close? if io.Copy errors `dest`will not be closed...

File: commands/bootstrap/resources.go [hunk 2]
```diff
@@ -163,7 +172,6 @@ func getVersionFromTOSCATypes(path string) string {
 			fmt.Println("Failed to open tosca types zip in bundled resources", err)
 			return version
 		}
-		defer src.Close()
```
Comment (loicalbertin) [inline commands/bootstrap/resources.go]: Same as above

File: commands/bootstrap/resources.go (hunk 1 above)
Comment (stebenoist) [reply commands/bootstrap/resources.go]: I removed the defer from the loop. The close is below

Comment (stebenoist) [reply commands/bootstrap/resources.go]: But yes, it's a bad idea

Comment (loicalbertin) [reply commands/bootstrap/resources.go]: I missed the loop when reviewing...
If there is a defer in a loop that sounds like it should be done into a function (even an anonymous one)

File: commands/bootstrap/resources.go [hunk 3]
```diff
@@ -176,6 +184,11 @@ func getVersionFromTOSCATypes(path string) string {
 			version = match[1]
 			return version
 		}
+		err = src.Close()
```
Comment (loicalbertin) [inline commands/bootstrap/resources.go]: When matching close will never be called.

Comment (stebenoist) [reply commands/bootstrap/resources.go]: I changed this !

**Pull request merged**